src/api/json/catalog.json

```diff
@@ -6620,6 +6620,24 @@
       "description": "Spice.ai OSS Spicepod Manifest file",
       "fileMatch": ["spicepod.yml", "spicepod.yaml"],
       "url": "https://raw.githubusercontent.com/spiceai/spiceai/trunk/.schema/spicepod.schema.json"
+    },
+    {
+      "name": "CRS WAF test file",
+      "description": "Definition of a test for verifying WAF behavior",
+      "fileMatch": ["**/tests/**/*.y?ml"],
```

Comment:
This is a very broad file match. Is there a way to narrow it down to avoid false positive for all `.yml` files inside `tests` folders?

Reply:
There's no convention for users for how to name their files. The CRS project names the tests after rules, e.g., `123456.yaml`, but I don't think matching such a pattern is supported, is it? I also saw a couple of similarly broad matches.

If your issue is more with the `tests` substring, we could probably change that, maybe to `waf-tests`.

Reply:
@theseion Yeah `waf-tests` works! I think you also might want to convert the single `*.ya?ml` into two entries with different file extensions, since each entry is a glob and not a regex.

Reply:
Right, that should have been `*y?ml`. That should work I suppose?

Reply:
I think any use of `?` here would be unusual or improper - I'd recommend looking at other entries that match YAML files.

Reply:
Yes, of course 🤦.

Reply:
I've changed the extension to `.waft` and updated the glob.